pandas/core/matrix.py

```diff
@@ -1 +1,3 @@
+# flake8: noqa
+
```

Comment:
should slate for removal, or at least start raising a deprecation warning when people import this file